src/clauseNums.ts

```diff
@@ -10,7 +10,8 @@ export default function iterator(spec: Spec): ClauseNumberIterator {
   let inAnnex = false;
   let currentLevel = 0;
   let hasWarnedForExcessNesting = false;
-  const MAX_LEVELS = spec.opts.maxClauseDepth ?? Infinity;
+  // Ecma house style calls for a maximum of 5 levels of clause division
+  const MAX_LEVELS = spec.opts.maxClauseDepth ?? 6;
```

Comment:
```suggestion
  const MAX_LEVELS = spec.opts.maxClauseDepth ? (spec.opts.maxClauseDepth || Infinity) : 6;
```

This would map `0` to Infinity, which seems reasonable for an option that we accept over the command line.